### PR TITLE
Alert component

### PR DESCRIPTION
**Pull request by avindhawale**
In alert.html file, main content starts from line number 110 to 205.

### CODE REVIEW

**Comment by linkinsunil** [comment]
Good to merge

File: views/components/avatar.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+              </div>
+            </div>
+            <p>
+              <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2509%2509%2509%2509%2509%253C%21--%2520Avatar%2520--%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-xs%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-sm%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-md%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-lg%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Favatar.svg%2522%2520class%253D%2522avatar%2520av-xl%2522%2520%252F%253E%250A"
+                style="
+                    width: 656px;
+                    height: 316px;
```
Comment (Hnh013) [inline views/components/avatar.html]: a suggestion : make a class if feasible , to give styles to i-frame tag, rest all looks good ! code structure is well

File: views/components/avatar.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            <h3>Avatar square</h3>
+            <p>
+              You can achieve square avatar by just adding
+              <code>av-sq</code> class as shown below.
+            </p>
+            <div class="demo-component-wrapper m-5">
+              <div class="demo-component-avatar-wrapper">
+                <!-- Avatar Square-->
+                <img src="/assets/images/avatar.svg" class="avatar av-xs av-sq" />
+                <img src="/assets/images/avatar.svg" class="avatar av-sm av-sq" />
```
Comment (Hnh013) [inline views/components/avatar.html]: suggestion : if feasible, use alt attribute with img tag for increasing accessibility, not mandatory though

**Review by Hnh013 — APPROVED** [review]
few suggestions I've added , rest all is good

File: views/components/avatar.html (hunk 2 above)
Comment (avindhawale) [reply views/components/avatar.html]: done.

**Pull request merged**